modules/boringssl/0.0.0-20240427-e2d7f2d/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/google/boringssl/archive/e2d7f2d625257b2c250ea2f18d3f832add5b43bf.tar.gz",
```

Comment:
Please do not merge this as-is. Going to see what the rest of the team things, but we may require that https://boringssl-review.googlesource.com/c/boringssl/+/68127 land first, so we can properly support this.

Reply:
We ended up going with that option, yeah. Could you update to a slightly newer version? No need to run our tests on BCR presubmit if that's not what BCR normally does, but it should be possible for folks using BoringSSL to run our tests, if needed, to diagnose problems (e.g. with their toolchain).

Reply:
Yeah let's try this and see how it works. Thanks!